src/main/java/no/cantara/aws/sqs/AmazonSQSSecureClient.java

```diff
@@ -238,7 +230,19 @@ public final SendMessageResult sendMessage(String queueUrl, String messageBody)
 
     @Override
     public final SendMessageBatchResult sendMessageBatch(SendMessageBatchRequest sendMessageBatchRequest) {
-        throw new NoSuchMethodError("TBI");
+        SendMessageBatchResult result = super.sendMessageBatch(sendMessageBatchRequest);
+
+        String queueUrl = sendMessageBatchRequest.getQueueUrl();
+        String queueName = queueUrl.substring(queueUrl.lastIndexOf('/') + 1);
+        if(queueNamesWithSnsNotification.contains(queueName)) {
+            for (SendMessageBatchResultEntry entry : result.getSuccessful()) {
+                CreateTopicResult topic = AWS_SNS_CLIENT.createTopic(queueName);
+                String topicArn = topic.getTopicArn();
```

Comment:
Her kan vell `AWS_SNS_CLIENT.createTopic(queueName)` flyttes ut av for-loopen? 🙂

Reply:
Ja, det burde den kanskje. Vi bruker ikke denne funksjonaliteten, så jeg vet ikke hva den egentlig gjør... Det er greit å sende flere meldinger på denne topicen etterhverandre antar jeg?

Reply:
Flytta det ut hvert fall :)

Reply:
Antar at det burde funke 😄